app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -156,17 +200,76 @@ public void updateIfNeeded() {
             changed = true;
         }
 
+        /* Section - options */
         // Disable weak hash benchmark for faster startup.
         // https://github.com/syncthing/syncthing/issues/4348
         Element options = (Element) mConfig.getDocumentElement()
                 .getElementsByTagName("options").item(0);
         changed = setConfigElement(options, "weakHashSelectionMethod", "never") || changed;
 
+        /* Check if we have to dismiss any specific "unackedNotificationID" */
+        /* Dismiss "fsWatcherNotification" according to https://github.com/syncthing/syncthing-android/pull/1051 */
+        if (getConfigElement(options, "unackedNotificationID").contains("fsWatcherNotification")) {
+            Log.i(TAG, "Remove 'fsWatcherNotification' from 'unackedNotificationID' node.");
+            changed = setConfigElement(options, "unackedNotificationID", getConfigElement(options, "unackedNotificationID").replace("fsWatcherNotification", "")) || changed;
+        }
```

Comment:
I expect this still doesn't do what you expect it to.
The resulting XML is like this:

```
<options>
   <foo>11</foo>
   <baz>Nuff</baz>
   <unackedNotificationID>notification1</unackedNotificationID>
   <unackedNotificationID>notification2</unackedNotificationID>
</options>
```

So I am not sure what it would be removing in this case.

Reply:
I verified the setConfigElement....replace working correctly just removing the one desired fsWatcher word from the unackedNotificationID string in an earlier debug build containing this code snippet. That can be explained by looking at setConfigElement. First, the "if" clause is true, because the word was found - that is when unackedNotificationID exists and contains a string - so setConfigElement will not add a new node instead it will update the existing unackedNotificationID node string. All fine :)

Reply:
To me it looks like you just remove the `fsWatcherNotification` string within `unackedNotificationID` node. What you should do is remove the entire `unackedNotificationID` node that contains `fsWatcherNotification`.

Reply:
But this only applies for the first element which might not always be the case and also leaves a tag with an empty string which is nit what we want.

Reply:
@imsodin I had a code suggestion to remove the whole node as advised, but @AudriusButkevicius did not agree this is the best way to do it. Removing just the one word can leave the empty string. If this is not okay, we can make a consent on deleting the whole node if it would end up empty.

Reply:
Improved that part detecting if removing or cleaning a word fist best and doing it.

Reply:
Corrected by commit https://github.com/syncthing/syncthing-android/pull/1051/commits/896829c21907d65e508a815d2953c3627b84c9b7